client/main.c

```diff
@@ -58,16 +58,15 @@
 #include "main.h"
 
 enum {
-	OPT_HELP,
-	OPT_VERSION,
-	OPT_CONFIGFILE,
-	OPT_DEBUG,
-	OPT_LOG_LEVEL,
-	OPT_LOG_TARGET,
-	OPT_SYSTEMD,
-
-	OPT_DRYRUN,
-	OPT_ROOTDIR,
+	OPT_HELP	= 'h',
+	OPT_VERSION	= 'v',
+	OPT_CONFIGFILE	= 'c',
+	OPT_DEBUG	= 'd',
+	OPT_LOG_LEVEL	= 'l',
+	OPT_LOG_TARGET	= 't',
```

Comment:
Everywhere else, we use `-t <timeout>`. Also remove the `OPT_LINK_TIMEOUT` enum, it's never used.

Reply:
Changet OPT_LOG_TARGET to L